Makefile.js

```diff
@@ -13,41 +13,23 @@
 
 require("shelljs/make");
 
-var nodeCLI = require("shelljs-nodecli");
+const nodeCLI = require("shelljs-nodecli");
 
 //------------------------------------------------------------------------------
 // Data
 //------------------------------------------------------------------------------
 
-var NODE_MODULES = "./node_modules/",
+const NODE_MODULES = "./node_modules/",
     TEMP_DIR = "./tmp/",
     BUILD_DIR = "./build/",
 
     // Utilities - intentional extra space at the end of each string
-    MOCHA = NODE_MODULES + "mocha/bin/_mocha ",
+    MOCHA = `${NODE_MODULES}mocha/bin/_mocha `,
 
     // Files
     MAKEFILE = "./Makefile.js",
-    /* eslint-disable no-use-before-define */
-    JS_FILES = find("lib/").filter(fileType("js")).join(" ") + " espree.js",
-    TEST_FILES = find("tests/lib/").filter(fileType("js")).join(" ");
-    /* eslint-enable no-use-before-define */
-
-//------------------------------------------------------------------------------
-// Helpers
-//------------------------------------------------------------------------------
-
-/**
- * Generates a function that matches files with a particular extension.
- * @param {string} extension The file extension (i.e. "js")
- * @returns {Function} The function to pass into a filter method.
- * @private
- */
-function fileType(extension) {
```

Comment:
Was this unused?

Reply:
it was used before the change, but I replaced it with `lib/*.js`.

Reply:
Ah, I see - missed that. Thanks :+1: